### ISSUE TITLE
v2.6.0 Can't resolve 'loglevel' in webpack-dev-server/client

### ISSUE THREAD

**Issue by subblue** [webpack/webpack-dev-server#1000]
**Do you want to request a *feature* or report a *bug*?**
Bug

**What is the current behavior?**
Just installed the latest v2.6.0 release and am now getting the error `Can't resolve 'loglevel' in webpack-dev-server/client`

**If the current behavior is a bug, please provide the steps to reproduce.**

The dev server config bit:
```
{
      port:               3000,
      host:               0.0.0.0,
      quiet:              false,
      hot:                true,
      contentBase:       path.resolve(__dirname, 'src'),
      historyApiFallback: true,
      disableHostCheck:   true,
      stats:              {
        hash:    false,
        colors:  true,
        cached:  false,
        version: false
      }
    }
```

**Please mention your webpack and Operating System version.**
webpack v3.3.0, OSX 10.13 (High Sierra beta 3)

**Comment by ena1106** [comment]
Duplicated of #984

**Comment by VitorLuizC** [comment]
A workaround to solve it is installing loglevel.
```shell
npm i -D loglevel
```

**Comment by hstarorg** [comment]
This dependencie should be in `package.json` `dependencies`, but in 2.6.0, it's in `devDependencies`.
We can downgrade version or use `npm i -D loglevel` to resolve.

**Comment by SpaceK33z** [comment]
Sorry everyone, fixed in webpack-dev-server@2.6.1
